### PR TITLE
Add `Contribute to the osu! News Feed` news post

### PR DESCRIPTION
**Pull request by 0x84f**
No specific date/time when this needs to go up, just posting so we can get reviews going

- [x] Redate/retime
- [x] change names of all relevant folders if post title is changed
- [x] ping me to enable responses on the form!!!!!! (Eph and Wala have perms as well)

### CODE REVIEW

File: news/2023/2023-07-08-writing-callout.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Knowledge of the Markdown language
+- Basic knowledge of GitHub
+- Knowledge of the [news styling criteria](/wiki/News_styling_criteria)
+- A good knowledge of the English language & its grammar
+
+## More tournament content on the front page
+
+For the longest time tournaments have been underrepresented on the front page, and we aim to change that going forward.
+
+Tournament hosts can now reach out to the [tournament committee](/wiki/People/Tournament_Committee) through email if they wish for their tournaments to be advertised, and after their review and discussion with the wiki & news team in the ``osu-news`` channel they will inform you with the final decision.
```
Comment (Blooshing) [inline news/2023/2023-07-08-writing-callout.md:36]:
```Suggestion
Tournament hosts can now reach out to the [tournament committee](/wiki/People/Tournament_Committee) through email if they wish for their tournaments to be advertised, and after their review and discussion with the wiki and news team in the ``osu-news`` channel they will inform you with the final decision.
```

File: news/2023/2023-07-08-writing-callout.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+If you are keen on sharing your writing with the community, the process is incredibly simple! Again, you can join the official Discord and share your drafts in the ``osu-news`` channel, where the wiki & news contributors can evaluate your work, give feedback and help you with the process of pushing your work to the front page!
+
+If you don't have any ideas currently, there are some already pinned in the channel which you can freely grab and work on.
+
+The only prerequisite is that you should be comfortable with constructive criticism and editorial feedback! We have listed additional points below that will make the process easier, but are not necessary and can be learned on the fly as you work on your project:
+
+- Knowledge of the Markdown language
+- Basic knowledge of GitHub
+- Knowledge of the [news styling criteria](/wiki/News_styling_criteria)
+- A good knowledge of the English language & its grammar
```
Comment (Blooshing) [inline news/2023/2023-07-08-writing-callout.md:30]:
```Suggestion
- A good knowledge of the English language and its grammar
```

File: news/2023/2023-07-08-writing-callout.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+layout: post
+title: "Contribute to osu!'s News Content"
+date: 2023-07-08 09:00:00 +0000
+---
+
+Ever wanted to contribute to osu!'s front page content? Now is the perfect time to do so!
+
+![](/wiki/shared/news/2023-07-08-writing-callout.md/banner.jpg)
+
+After last years' efforts to spice up the front page with a variety of news posts via the [content call forum post](https://osu.ppy.sh/community/forums/topics/1605337), which led up to [osu!monthly](/wiki/Community/osu!monthly) making a comeback for its [3rd season](/wiki/Community/osu!monthly#season-3-(2022–2023)) & a series of news posts by [0x84f](https://osu.ppy.sh/users/7944724), we wanted to spread the word further and make contributing more accessible to the community.
```
Comment (Blooshing) [inline news/2023/2023-07-08-writing-callout.md:11]:
```Suggestion
After last years' efforts to spice up the front page with a variety of news posts via the [content call forum post](https://osu.ppy.sh/community/forums/topics/1605337), which led up to [osu!monthly](/wiki/Community/osu!monthly) making a comeback for its [3rd season](/wiki/Community/osu!monthly#season-3-(2022–2023)) and a series of news posts by [0x84f](https://osu.ppy.sh/users/7944724), we wanted to spread the word further and make contributing more accessible to the community.
```

**Review by Blooshing — CHANGES_REQUESTED** [review]
leaving these changes here to for more consistency for formal writing.

File: news/2023/2023-07-08-writing-callout.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Are you interested in creative writing and want to place your content in front of hundreds of thousands of players, or just have an idea that you would like to see covered on the front page? Then feel free to read below!
+
+## Suggesting ideas
+
+If you have an idea that you think would be a good fit for the front page, you can now submit them via this anonymous [suggestion box](https://docs.google.com/forms/d/e/1FAIpQLSeuv9heFjBX_NPDzpMvV2ORaNllHtzWKRDmk5IqSMEWvc-SNw/viewform)! Once submitted, the responses will be reviewed and ideas that are deemed fit for the front page will be posted in the ``osu-news`` channel on the [official osu! Discord](https://discord.com/invite/ppy).
+
+## Writing for the front page
+
+If you are keen on sharing your writing with the community, the process is incredibly simple! Again, you can join the official Discord and share your drafts in the ``osu-news`` channel, where the wiki & news contributors can evaluate your work, give feedback and help you with the process of pushing your work to the front page!
```
Comment (0x84f) [inline news/2023/2023-07-08-writing-callout.md:21]:
```suggestion
If you are keen on sharing your writing with the community, the process is incredibly simple! Again, you can join the official Discord and share your drafts in the ``osu-news`` channel, where the wiki and news contributors can evaluate your work, give feedback and help you with the process of pushing your work to the front page!
```

**Pull request merged**